backend/Routes/index.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const auth = require('../Controllers/auth');
+const app = express();
+
+app.use(express.urlencoded({extended: true}));
+app.use(express.json());
+
+app.post('/login', auth.login);
+app.post('/signup', auth.signup);
+app.post(
+    '/users/:username/forgot_password',
```

Comment:
nit: How about using /auth/forgot_password as the endpoint? Keeping the URL same, so that we check token of the user only..

Reply:
ok